Description: Unneccessary shebang line in scripts/recon.pl
 Correct issue with unneccessary shebang line in scripts/recon.pl
Bug: http://sourceforge.net/tracker/?func=detail&aid=3473848&group_id=175965&atid=875350
Author: Robert James Clay <jame@rocasa.us>

--- a/scripts/recon.pl
+++ b/scripts/recon.pl
@@ -1,4 +1,3 @@
-#!/usr/bin/perl
 =pod
 
 =head1 NAME
